autoload/neomake/makers/ft/jade.vim

```diff
@@ -0,0 +1,18 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#jade#EnabledMakers()
+    let makers = []
+    if neomake#utils#Exists('jade-lint')
```

Comment:
See https://github.com/benekastah/neomake/blob/master/autoload/neomake/makers/ft/javascript.vim for a typical example of this function.